build.gradle

```diff
@@ -210,6 +210,10 @@ def determineCommitId() {
 apply from: file('wrapper.gradle')
 
 if (repo != null) {
+  ext {
+    ignoreRatFailure = project.hasProperty('ignoreRatFailure') ? project.ignoreRatFailure.toBoolean() : false
```

Comment:
Can you relocate this with other property handling? (Along with userMaxForks, etc)